assets/javascripts/openqa.js

```diff
@@ -626,15 +626,21 @@ function renderHttpUrlAsLink(value) {
     value = String(value);
   }
   const urlRegex = /https?:\/\/[^\s,]*/g;
+  const arrayOfUrlsRegex = /^(https?:\/\/[^\s,]*)(\s+https?:\/\/[^\s,]*)+$/g;
```

Comment:
seems to work. would you mind to add a comment that this is intent to match arrays of URL *with space*. And I would appreciate it if you could write this also in the commit description.

Reply:
I am gonna resolve this as comment added + commit was updated. However, I'm not sure about the commit wording...